tests/Fixtures/Maker/expected/can_create_factory_for_not_persisted_class.php

```diff
@@ -41,7 +41,7 @@ protected function getDefaults(): array
             'intMandatory' => self::faker()->randomNumber(),
             'propertyWithoutType' => null, // TODO add value manually
             'someMandatoryPropertyWithUnionType' => null, // TODO add value manually
-            'someOtherObjectMandatory' => null, // TODO add Zenstruck\Foundry\Tests\Fixtures\Object\SomeOtherObject value manually
+            'someOtherObjectMandatory' => SomeOtherObjectFactory::new(),
```

Comment:
at first I thought "there is a bug, the factory class is not imported", but both `SomeOtherObjectFactory` and this one are in the same namespace :sweat_smile: